workshops/python_neural_network/README.md

```diff
@@ -0,0 +1,229 @@
+---
```

Comment:
This seems like an old version of the workshop? I think you should remove this directory entirely and just rename the other one to `python_neural_network` or something